apps/aestratum/priv/Payout.aes

```diff
@@ -0,0 +1,11 @@
+contract Payout =
+
+  public function payout(xs : list((address, int))) : int =
```

Comment:
Could you please check that compilation of this contract with intended supported contract works. Is it hooked in test with versioned compiler? (rebar3 test profile has some compilers.)

Reply:
It's not hooked, as such thing is rare. We could eventually do this but it certainly shouldn't block the merge I think...